client/app/scripts/superdesk-ingest/module.js

```diff
@@ -79,8 +80,12 @@ define([
         $rootScope.currentModule = 'ingest';
 
         this.fetchItems = function(criteria) {
+            $scope.loading = true;
             api.ingest.query(criteria).then(function(items) {
                 $scope.items = items;
+                $scope.loading = false;
+            }, function(response) {
```

Comment:
fyi you can use [promise.finally](https://docs.angularjs.org/api/ng/service/$q#the-promise-api)